easybuild/easyblocks/p/pgi.py

```diff
@@ -121,6 +121,14 @@ def install_step(self):
 
         cmd = "%s -x %s -g77 /" % (filename, install_abs_subdir)
         run_cmd(cmd, log_all=True, simple=True)
+        
+        # If an OS libnuma is NOT found, makelocalrc creates symbolic links to libpgnuma.so
+        # If we use the EB libnuma, delete those symbolic links to ensure they are not used
+        if get_software_root("numactl"):
```

Comment:
you'll need to import this function too, i.e. add this on top (in alphabetical order compared to the ones already there):

```python
from easybuild.tools.modules import get_software_root
```